stories/assets/scss/_mixins.scss

```diff
@@ -69,6 +69,17 @@
   @include transform(skew(#{$x}deg, #{$y}deg));
 }
 
+// Arrow{
+@mixin arrowCarousel() {
+  background: $color-gray-700;
+  border: none;
+  border-radius: 50%;
+  cursor: pointer;
+  height: 2.784rem;
+  margin: 0 0.4rem;
+  width: 2.784rem;
+}
```

Comment:
@Shamimkhan321 if we are not using the parameter then we can create it placeholder instead of mixin